mix.exs

```diff
@@ -12,7 +12,7 @@ defmodule Timex.Mixfile do
   end
 
   def application do
-    [included_applications: [:tzdata]]
+    [included_applications: [:tzdata], timezone: nil]
```

Comment:
Is this necessary to add? `get_env` and `put_env` should work fine without listing it here.